addon/services/user-activity.js

```diff
@@ -34,6 +34,7 @@ export default Service.extend(Evented, {
   },
 
   _listen(eventName) {
+	if (!self.window.addEventListener) { return; }
```

Comment:
I'm unsure why you need to do `self.window`...is `window` not available in fastboot?